python/paddle/v2/__init__.py

```diff
@@ -42,6 +43,12 @@
 
 def init(**kwargs):
     args = []
+    # NOTE: append arguments if they are in ENV
+    for ek, ev in os.environ.iteritems():
+        if ek.startswith("PADDLE_"):
+            args.append('--%s=%s' % (ek.replace("PADDLE_", "").lower(), str(ev)))
```

Comment:
lstrip will see `"PADDLE_"` as a iterative of chars, and remove the chars not string.